components/ui/toast.tsx

```diff
@@ -0,0 +1,129 @@
+'use client';
+
+import * as ToastPrimitives from '@radix-ui/react-toast';
+import { cva, type VariantProps } from 'class-variance-authority';
+import { Cross2Icon } from '@radix-ui/react-icons';
```

Comment:
Um, we're not using this lib. The build is also failing since this isn't added to `package.json` either.

Reply:
wait why was I able to even commit then, wth